legacy/src/main/java/org/opensearch/sql/legacy/executor/multi/MinusExecutor.java

```diff
@@ -49,6 +57,7 @@ public class MinusExecutor implements ElasticHitsExecutor {
   private String[] fieldsOrderFirstTable;
   private String[] fieldsOrderSecondTable;
   private String seperator;
+  private PointInTimeHandler pit;
```

Comment:
If all executor depends on PIT, `getResponseWithHits` and others like `Client`, probably we can move all common fields to `ElasticHitsExecutor` and make it abstract class?